### PR TITLE
pekko 1.1.0-M1 release

### PR DESCRIPTION
**Pull request by pjfanning**
vote has passed

### CODE REVIEW

File: project/build.properties [hunk 1]
```diff
@@ -1 +1 @@
-sbt.version=1.9.9
+sbt.version=1.10.0
```
Comment (raboof) [inline project/build.properties:1]: weird to have this in this PR?

**Review by raboof — APPROVED** [review]

File: project/build.properties (hunk 1 above)
Comment (pjfanning) [reply project/build.properties:1]: just some times seems easier to not do separate PRs for small things when you are already working on a PR in the area

Comment (pjfanning) [reply project/build.properties]: reverted

File: src/main/paradox/download.md [hunk 2]
```diff
@@ -15,10 +15,15 @@ These jars relate to the source downloads you find linked below.
 
 * [apache-pekko-1.0.2-incubating-src-20231115.tgz](https://www.apache.org/dyn/closer.lua/incubator/pekko/1.0.2/apache-pekko-1.0.2-incubating-src-20231115.tgz) signature: ([asc](https://downloads.apache.org/incubator/pekko/1.0.2/apache-pekko-1.0.2-incubating-src-20231115.tgz.asc)) checksum: ([sha512](https://downloads.apache.org/incubator/pekko/1.0.2/apache-pekko-1.0.2-incubating-src-20231115.tgz.sha512))            
 
-There is also a milestone release and is aimed at testing some new support for users who want to add Pekko nodes to an existing Akka cluster. This support is experimental. This release should not be used in production:
+There are also 2 milestone releases. These releases should not be used in production.
```
Comment (He-Pin) [inline src/main/paradox/download.md:18]: Milestones

**Review by He-Pin — APPROVED** [review]
Lgtm

**Pull request merged**